src/rust/ide/src/controller/searcher/action.rs

```diff
@@ -41,6 +43,7 @@ impl Action {
                 completion.name.clone()
             }
             Self::Example(example) => format!("Example: {}", example.name),
+            Self::CreateNewProject => "Create New Project".to_owned(),
```

Comment:
```suggestion
            Self::CreateNewProject => "Create a New Project".to_owned(),
```
but then it looks like PoKeMoN script.
Perhaps just `New Project`?